### DISCUSSION TITLE
I wanted to use security onion to get logs from  microsoft azure and generate alerts

### DISCUSSION THREAD

**Discussion by Jaideep555** [Security-Onion-Solutions/securityonion#13395] [Unsupported Versions]
Basically i will use a vm on azure with security onion. I have configured azure resources to send logs to the storage account. Now i want security onion to get those logs and generate alerts. Can anyone tell me how to get it done? or any idea?. Any help would be greatly appreciated.
I am very new to this so i am pretty clueless.

**Comment by cm-ops** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/1720

Can you give us a little more background please? What type of node, what logs are you looking to ingest, etc.

**Reply by Jaideep555** [reply]
hello, first, thanks a lot for replying and also sorry for late reply. My priority is to get logs from Entra ID, VMs (hosted on azure), storage account logs, SQL database logs, key-vault logs and data-factory logs. Also, I used standalone deployment

**Reply by cm-ops** [reply]
Have you tried enabling the Azure integration https://github.com/elastic/integrations/tree/main/packages/azure?

Also, see https://docs.securityonion.net/en/2.4/third-party-integrations.html

If there are logs not covered by the integration you can always try a custom integration.

**Reply by Jaideep555** [reply]
I have read these already, thx though. The issue was i wanted to reduce the cost. So, can i import logs from sources other than event hubs as that would be much cheaper for me? Like from azure storage directly, or any other way?

**Reply by cm-ops** [reply]
There is an Elastic Azure Blob Storage connector client, but that is a paid license feature through Elastic.

How were you thinking of connecting to the storage?

**Reply by Jaideep555** [reply]
That is what i needed advice about. I wanted to get logs from azure storage instead of hubs but couldn't find any related articles or any posts. I guess i can try going through info about the blob storage connector you mentioned. thx a lot
